drivers/sensor/bq2415x/bq2415x.h

```diff
@@ -0,0 +1,153 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * bq2415x charger driver
+ *
+ * Copyright (C) 2011-2013  Pali Rohár <pali@kernel.org>
+ * Copyright (C) 2021  Filip Primozic <filip.primozic@greyp.com>
+ */
+
+#ifndef BQ2415X_CHARGER_H
+#define BQ2415X_CHARGER_H
+
+#include <stdint.h>
+#include <kernel.h>
+
+#define BQ241X_VENDER_CODE 0x0A /* Default vender code (in binary) */
+
+/* timeout, in seconds, for resetting chip timer */
+#define BQ2415X_TIMER_TIMEOUT 10
+
+#define BQ2415X_REG_STATUS 0x00
+#define BQ2415X_REG_CONTROL 0x01
+#define BQ2415X_REG_VOLTAGE 0x02
+#define BQ2415X_REG_VENDER 0x03
+#define BQ2415X_REG_CURRENT 0x04
+
+/* reset state for all registers */
+#define BQ2415X_RESET_STATUS BIT(6)
```

Comment:
Can you indent it please
```
#define BQ2415X_RESET_CONTROL\t---->(BIT(4) | BIT(5))
#define BQ2415X_RESET_VOLTAGE\t---->(BIT(1) | BIT(3))
```

Reply:
This is formatted by the checkpatch tool. If I change it to the suggested, tool will revert it back.